devtools_wp/cypress/support/wp_cli_helpers.js

```diff
@@ -15,6 +15,10 @@ function createWooEntity(entity, params) {
   cy.exec(`docker-compose exec -u www-data -T web wp wc ${entity} create ${options}`)
 }
 
+function setDripAccount(account_id){
+  cy.exec(`docker-compose exec -u www-data -T web wp option update account_id "${account_id}"`)
```

Comment:
Unrelated to this PR, but this is making me realize that `account_id` as a settings identifier is global to all plugins installed on the WP site. I wonder whether we should change this to `drip_account_id` or something like that… In a different PR, of course.